spec/fixtures/ssl/request.pem

```diff
@@ -1,10 +1,10 @@
 Certificate Request:
     Data:
-        Version: 3 (0x2)
```

Comment:
Could you please document how you generated the fixtures?

Reply:
> Could you please document how you generated the fixtures?

Yes. I had meant to, and then forgot. Working on it.

Reply:
Okay, I pushed an update with a script which can generate the ones in question.  I wasn't prepared to take on the whole ssl/fixtures directory this time around.